src/chia_log/handlers/condition_checkers/non_skipped_signage_points.py

```diff
@@ -34,31 +32,15 @@ def check(self, obj: FinishedSignagePointMessage) -> Optional[Event]:
         )
 
         if not valid:
-            # Reset state when we receive non-valid order of signage points
-            # this ensures that we aren't sending any wrongly calculated skips
-            self._last_signage_point_timestamp = None
-            self._last_signage_point = None
             return None
 
         # To reduce notification spam, only send notifications for skips larger than 1
-        # or for multiple individual skips within 1 hour
         if skipped == 1:
-            logging.info(f"Detected {skipped} skipped signage point.")
-            if self._last_skip_timestamp:
-                minutes_since_last_skip = (datetime.now() - self._last_skip_timestamp).seconds // 60
-                if minutes_since_last_skip > 60:
-                    logging.info("No other skips in the last 60 minutes. Can be safely ignored.")
-                else:
-                    message = "Experiencing networking issues? Skipped 2+ signage points in the last hour."
-                    logging.warning(message)
-                    event = Event(
-                        type=EventType.USER,
-                        priority=EventPriority.NORMAL,
-                        service=EventService.FULL_NODE,
-                        message=message,
-                    )
-
-        if skipped >= 2:
+            logging.info(
+                f"Detected {skipped} skipped signage point."
```

Comment:
Consider moving to debug?

Reply:
Maybe in the future 👍  for now I'd prefer to have it visible just like the 30+ second gaps.